taskfiles/deps/installs/mongocxx.yaml

```
@@ -0,0 +1,24 @@
+            - "-DBUILD_SHARED_AND_STATIC_LIBS=ON"
+            - "-DBUILD_SHARED_LIBS_WITH_STATIC_MONGOC=ON"
+            - "-DCMAKE_BUILD_TYPE=Release"
+            - "-DCMAKE_INSTALL_MESSAGE=LAZY"
+            - "-DENABLE_EXAMPLES=OFF"
+            - "-DENABLE_UNINSTALL=OFF"
+          INSTALL_TASK_NAME: "mongocxx"
+          LIB_NAME: "mongocxx"
+          TARBALL_SHA256: "19dff3cf834a3e09229260f22a0325820a7e30c78b294db91794dd934776b33a"
+          TARBALL_URL: "https://github.com/mongodb/mongo-cxx-driver/releases/download/{{.VERSION}}\
+          /mongo-cxx-driver-{{.VERSION}}.tar.gz"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)